pytest/test_cli_features.py

```diff
@@ -0,0 +1,32 @@
+import os
+import json
+import pytest
+import importlib.util
+
+def get_test_directories():
+    test_dirs = []
+    for root, dirs, files in os.walk(os.getcwd()):

```

Comment:
I suggest using `glob.glob('complexity-*')` instead of os.walk(). Just look for files directly under pytest/ called "complexity-*". That simplifies things.